src/test/rgw/rgw_multi/tests_ps.py

```diff
@@ -166,89 +166,6 @@ def close(self):
             worker.join()
 
 
-# AMQP endpoint functions
```

Comment:
you cannot remove that. this is still needed for the pubsub+push tests (tests that use the pubsub zone, but still push to kafka/amqp)

Reply:
Okay so every function should be there as it was?

Reply:
i don't see what option you have. you cannot run these tests otherwise...

Reply:
Yeah that's true.